CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This file is used to list changes made in each version of the haproxy cookbook.
 
+## [v8.1.2] (2019-10-10)
```

Comment:
Please make this heading unreleased. I won't be making a release for this as it's not changing the cookbook code
```suggestion
## Unreleased
```

Reply:
Done